lection4/task2.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+let age_1 = '12';
+const age_2 = 18;
+const age_3 = 60;
+
+if (typeof age_1 === 'string') {
+    age_1 = +age_1;
```

Comment:
No checks for NaN values. Please add required checks

Reply:
for what?
if age_1 =  NaN -> Technical works